src/injector.js

```diff
@@ -159,6 +119,47 @@ export default class Injector {
     }
   }
 
+  injectIntoDOM(dependencies, idx = 0, type = null) {
+    if (idx >= dependencies.length) { return; }
+
+    // inject order: explicitly provided < cached in local storage < printed
+    // raw only as fallback if printed fails
+    type = (dependencies[idx][type] && type) || (dependencies[idx]['cached'] && 'cached') || (dependencies[idx]['printed'] && 'printed');
```

Comment:
nope ... DOM.js returns either `{ cached: ... }` or `{ raw: ..., printed: ... }` .. so if `dependencies[idx]['cached']` is not set this evaluates to false and `"cached"` never gets evaluated ... code execution continues at the next statement that might be truthy which is `dependencies[idx]['printed']`